docs/insomnia/requests.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: article-detail
+title:  Requests
+category: "Requests and Responses"
+category-url: requests-and-responses
+---
+
+Insomnia supports sending requests via [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), [gRPC](https://grpc.io/docs), and [GraphQL](https://graphql.org).
+
+To learn how to repurpose parts of your request, see
+[Environment Variables](/insomnia/environment-variables).
+
+## Send an HTTP method request
+
+Send an HTTP method request in a Document or Collection. The following instructions
+assume you're starting with an empty Document or Collection.
+
+In addition to standard HTTP methods, Insomnia supports [Custom HTTP Methods](#custom-http-methods).
+
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select a method from the dropdown.
+   1. Click **Create**.
+1. Enter the API URL in the request URL input.
+1. To add authentication, go to the **Auth** tab. Use the **Auth** dropdown to select your authentication method.
+1. When you have entered your request URL and authentication, click **Send**.
+
+See [Request Options](#request-options) to customize your request.
+
+## Send a gRPC request
+
+Send a gRPC method request in a Document or Collection. The following instructions
+assume you're starting with an empty Document or Collection.
+
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select **gRPC** from the dropdown.
+   1. Click **Create**.
+1. You'll be taken to a **Select Proto File** modal.
+1. Click **Add Directory** or **Add Proto File**.
+1. Add your directory or upload your Proto File. Insomnia will automatically detect the streaming types and modify the first tab as such.
+1. Enter your API URL and click **Start**.
```

Comment:
The button could be "Start" or "Send", depending on the streaming type. You can get an example protofile from `https://grpcb.in/` to experiment with (the `hello.proto` one is great). Alternatively, it's also part of the Insomnia Testing project, inside the gRPC collection.

Also, users need to select a method to use from the dropdown next to the URL.

![image](https://user-images.githubusercontent.com/4312346/146468060-aada048f-b3fc-403f-9df5-b42e083e096a.png)

Reply:
Is it worth noting how to send client streaming and Bidirectional streaming requests? Because these have extra steps: https://www.loom.com/share/1a287dcb26e44b01a326d3eafb840d84

Reply:
Yes, let's do that in a future iteration.